src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,153 @@
+"use client";
+import Card from "@/components/card";
+import Input from "@/components/input";
+import { apiWithCredentials } from "@/lib/api";
+import { useAuthStore } from "@/stores/authStore";
+import { zodResolver } from "@hookform/resolvers/zod";
+import axios from "axios";
+import Link from "next/link";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import z from "zod";
+
+const formSchema = z.object({
+  email: z.email(),
+  password: z.string()
+})
+
+type FormSchema = z.infer<typeof formSchema>
+
+export default function Login() {
+  const { register, handleSubmit, formState: { errors } } = useForm<FormSchema>({
+    resolver: zodResolver(formSchema)
+  });
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const router = useRouter();
+
+  const onSubmit: SubmitHandler<FormSchema> = async (data) => {
+    setLoading(true);
+
+    try {
+      const res = await apiWithCredentials.post("/auth/sign_in", data);
+      const { access_token } = res.data;
+
+      useAuthStore.getState().setToken(access_token);
+      router.push("/");
+    } catch (error) {
+      if (axios.isAxiosError(error)) {
+        if (error.status === 401) {
+          setError("Incorrect email or password");
+        } else {
+          setError(error.message);
```

Comment:
```suggestion
          setError("Login unavailable at the moment. Please try again shortly.");
```

(this will have to be localized in the future)